scripts/setup-m1-builder.sh

```diff
@@ -17,7 +17,7 @@ sudo chmod g+rwx /usr/local/lib/
 
 brew update # Needed to sidestep bintray brownout
 # Usually we install python 3.7 for building wheels, but 3.7 isn't supported on m1 macs
-brew install opam pkg-config coreutils python pcre
+brew install opam pkg-config coreutils python pcre libev
```

Comment:
would be better to call make, to avoid defining at yet another place a list of package to install

Reply:
will do